tests/services/test_services.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020-2021 CERN.
+# Copyright (C) 2020-2021 Northwestern University.
+#
```

Comment:
Just as an *example*, in this case is correct. Because the file was touched by NU :) and you are moving content created also by them to this new file so you have to keep it as is. THIS IS CORRECT.